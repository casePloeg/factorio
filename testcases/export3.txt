run_script "/Users/caseploeg/factorio/factorio-cli/scripts/startup.txt" > /dev/null
set debug true
alias create lprod prod "|" less
alias create prodl lprod
alias create raw prod "|" grep -E "(ore|plate)"
alias create init @ scripts/factory.txt
alias create export history -s ">" export.txt
clear
mine stone 20
mine iron-ore 20
mine copper-ore 20
place stone-furnace iron-plate
place burner-mining-drill iron-ore
craft stone-furnace 4
inventory
wish burner-mining-drill
craft burner-mining-drill
craft copper-ore 20
mine copper-ore 20
mine stone 20
craft burner-mining-drill
place burner-mining-drill copper-ore
place stone-furnace copper-plate
inventory
craft burner-mining-drill
inventory
place burner-mining-drill iron-ore
place stone-furnace iron-plate
inventory
mine stone 20
inventory
craft burner-mining-drill 2
inventory
craft burner-mining-drill
place burner-mining-drill iron-ore
place stone-furnace iron-plate
craft stone-furnace 8
craft stone-furnace 5
place stone-furnace iron-plate 3
inventory
place stone-furnace copper-plate
inventory
craft burner-mining-drill
mine stone 30
craft burner-mining-drill
place burner-mining-drill stone
inventory
craft burner-mining-drill 4
place burner-mining-drill iron-ore 4
inventory
craft stone-furnace
inventory
craft stone-furnace 3
inventory
wish automation-science-pack
inventory
craft automation-science-pack
inventory
craft automation-science-pack 10
wish automation
craft automation-science-pack 4
inventory
craft automation-science-pack 6
research automation
craft assembling-machine-1
place assembling-machine-1 iron-gear-wheel
prod
inventory
craft burner-mining-drill 4
place burner-mining-drill copper-ore 4
inventory
craft asm1
craft assembling-machine-1
inventory
place assembling-machine-1 copper-cable
inventory
prodl
next
inventory
