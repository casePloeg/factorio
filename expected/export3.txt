{"assemblers": {"copper-cable:assembling-machine-1": 1, "iron-gear-wheel:assembling-machine-1": 1}, "current_items": {"assembling-machine-1": 0, "automation-science-pack": 1, "burner-mining-drill": 0, "copper-cable": 114, "copper-ore": 42, "copper-plate": 36, "iron-gear-wheel": 15, "iron-ore": 43, "iron-plate": 90, "stone": 38, "stone-furnace": 0, "water": 1000000000}, "current_recipes": ["assembling-machine-1", "automation-science-pack", "boiler", "burner-inserter", "burner-mining-drill", "copper-cable", "copper-plate", "electric-mining-drill", "electronic-circuit", "firearm-magazine", "inserter", "iron-chest", "iron-gear-wheel", "iron-plate", "iron-stick", "lab", "light-armor", "long-handed-inserter", "offshore-pump", "pipe", "pipe-to-ground", "pistol", "radar", "repair-pack", "small-electric-pole", "steam-engine", "stone-brick", "stone-furnace", "transport-belt", "wooden-chest"], "current_tech": ["automation"], "furnaces": {"copper-plate:stone-furnace": 2, "iron-plate:stone-furnace": 5}, "game_time": 332.25, "limited_items": {}, "miners": {"copper-ore:burner-mining-drill": 5, "iron-ore:burner-mining-drill": 7, "stone:burner-mining-drill": 1}}